src/molecules/formfields/FullDateField/full_date_field.module.scss

```diff
@@ -0,0 +1,3 @@
+.full-date-field {
```

Comment:
Do we need this file if there's no styles here?

Reply:
nope!